### ISSUE TITLE
Display class name with bounding box on ground truth

### ISSUE THREAD

**Issue by matt-sharp** [rafaelpadilla/review_object_detection_metrics#17]
When displaying the ground truth statistics would it be possible to display the class name with each bounding box?
This would greatly help with training set verification. Thanks!

**Comment by rafaelpadilla** [comment]
Hi @matt-sharp ,

In images with many bounding boxes, the class names would be overlapping each other. So, we prefer not to include the names.

But you can easily have this feature by changing one single line in the code.

Change [line 120 at details.py](https://github.com/rafaelpadilla/review_object_detection_metrics/blob/main/src/ui/details.py#L120), which reads:
`img = add_bb_into_image(img, bb, color=(0, 255, 0), thickness=2, label=None)`

to
`img = add_bb_into_image(img, bb, color=(0, 255, 0), thickness=2, label=bb._class_id)`

Best regards,
Rafael

**Comment by matt-sharp** [comment]
Fantastic, this works well. Thanks!
